scripts/docs.py

```diff
@@ -439,5 +430,51 @@ def get_sections(nav: list) -> Dict[Tuple[str, ...], str]:
     return sections
 
 
+def get_sections_all(nav: list) -> Dict[Tuple[str, ...], Set[str]]:
+    sections = {}
+    for item in nav:
+        if type(item) is str:
+            continue
+        elif type(item) is dict:
+            item_key = list(item.keys())[0]
+            sub_nav = item[item_key]
+            sections[(item_key,)] = set(
+                [sub for sub in sub_nav if isinstance(sub, str)]
+            )
+            sub_sections = get_sections_all(sub_nav)
+            for k, v in sub_sections.items():
+                new_key = (item_key,) + k
+                sections[new_key] = v
+    return sections
+
+
+def get_section_en_to_lang(
+    use_nav: list, lang_use_nav: list
+) -> Dict[Tuple[str, ...], Tuple[str, ...]]:
+    sections_set = get_sections_all(use_nav)
+    lang_sections_set = get_sections_all(lang_use_nav)
+    # Translate English section name, when translated section contains
+    # subset of files in original section.
+    section_en_to_lang = {section: section for section in sections_set.keys()}
+    for en_section, file_set in sections_set.items():
+        for lang_section, lang_file_set in lang_sections_set.items():
+            if lang_file_set.issubset(file_set):
+                section_en_to_lang[en_section] = lang_section
+
+    for en, lang in section_en_to_lang.items():
+        if len(en) >= 2 and en == lang:
```

Comment:
Reverse the condition and add the continue statement to decrease the indentation level.

```py
if len(en) < 2 or en != lang:
    continue

# your code goes here
```